automatic/gitextensions/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,37 @@
+$packageArgs = @{
+    packageName    = $env:ChocolateyPackageName
+    fileType       = 'msi'
+    file           = Get-ChildItem "$env:ChocolateyPackageFolder\tools\*.msi"
+    silentArgs     = '/quiet /norestart ADDDEFAULT=ALL REMOVE=AddToPath,Icons'
+    validExitCodes = @(0, 3010, 1641)
+}
+
+# the REMOVE parameter that is defined in silentArgs was obtained with the
+# following PowerShell snippet.
+# NB we do not let the installer add the GitExtensions directory to the PATH
```

Comment:
Remove comments

Reply:
why? they are useful to re-create the REMOVE parameter.

Reply:
There are all sorts of useful things one could put anywhere. The purpose of the code is not to educate people about the universe and everything else. You could add this among package `## Notes` for example.

From guidelines:

> Keep the package source files clean and remove obsolete or outdated code and unnecessary comments. Comment non-obvious code so that others can easily understand what it does.

Reply:
IMHO, the purpose of the comment is to help the maintainer not anything else. Anyways, I will move that comment into the README.